src/rpc/blockchain.cpp

```diff
@@ -2319,6 +2319,41 @@ UniValue scantxoutset(const JSONRPCRequest& request)
     return result;
 }
 
+static UniValue getlicensedminers(const JSONRPCRequest& request)
+{
+    if (request.fHelp || request.params.size() != 0)
+        throw std::runtime_error(
+            RPCHelpMan{"getlicensedminers",
+                "\nReturns the list of licensed miners.\n",
+                {},
+				RPCResult{
+				            "\"licenses\": [\n"
+				            "    {\n"
+				            "      \"height\": n,                  (numeric) Height of the last modifiaction of given license\n"
```

Comment:
spelling

Reply:
It should be `Height of the last modification of a given license`

Reply:
Done